src/drivers/magnetometer/hmc5883/hmc5883.cpp

```diff
@@ -640,18 +623,18 @@ HMC5883::ioctl(struct file *filp, int cmd, unsigned long arg)
 			/* adjust to a legal polling interval in Hz */
 			default: {
 					/* do we need to start internal polling? */
-					bool want_start = (_measure_ticks == 0);
+					bool want_start = (_measure_interval == 0);
 
 					/* convert hz to tick interval via microseconds */
-					unsigned ticks = USEC2TICK(1000000 / arg);
+					unsigned interval = (1000000 / arg);
```

Comment:
We might to update the comment above this line. I hadn't been looking at the comments previous to this, so there are probably more instances to update.

Reply:
Clarified a bit.